src/sonic-frr-mgmt-framework/templates/supervisord/critical_processes

```diff
@@ -0,0 +1,8 @@
+program:zebra
+program:staticd
+program:bfdd
+program:bgpd
+program:ospfd
```

Comment:
combine this with dockers/critical_process.j2

Reply:
Addressed